src/server.ts

```diff
@@ -34,11 +34,7 @@ const handleConnection = async (connection: Connection) => {
   app.use(adminBro.options.rootPath, router);
 
   // run express application on given port
-  app.listen(PORT, (err: any) => {
-    if (err) {
-      return console.error(err);
-    }
-
+  app.listen(PORT, () => {
```

Comment:
maybe we can can keep the error checking just in case?

Reply:
Since we are using the wrapper for the app express it isn't possible to use the same firm as we used to have. This now just receives the port and the function (without any params)